src/cljs/org/broadinstitute/firecloud_ui/page/workspace/monitor/common.cljs

```diff
@@ -0,0 +1,33 @@
+(ns org.broadinstitute.firecloud-ui.page.workspace.monitor.common
+  (:require
+    [dmohs.react :as react]
+    [org.broadinstitute.firecloud-ui.common :as common]
+    [org.broadinstitute.firecloud-ui.common.components :as comps]
+    [org.broadinstitute.firecloud-ui.common.icons :as icons]
+    [org.broadinstitute.firecloud-ui.common.style :as style]
+    [org.broadinstitute.firecloud-ui.common.table :as table]
+    [org.broadinstitute.firecloud-ui.page.workspace.monitor.workflow-details :as workflow-details]
+    [org.broadinstitute.firecloud-ui.endpoints :as endpoints]
+    ))
+
+
+(defn render-date [date]
```

Comment:
I did a quick code search and this is the only place I see this particular format. I don't want to make it more centrally located than its current usages dictate. If/When something else formats dates this way, that PR can move this into utils.